client/src/main/java/io/pravega/client/segment/impl/ConditionalOutputStreamImpl.java

```diff
@@ -149,7 +149,7 @@ RuntimeException handleUnexpectedReply(Reply reply, String expectation) {
         } else if (reply instanceof InvalidEventNumber) {
             InvalidEventNumber ien = (InvalidEventNumber) reply;
             throw Exceptions.sneakyThrow(new ConnectionFailedException(ien.getWriterId() + 
-                    " Got stale data from setupAppend on segment " + segmentId + " for ConditionalOutputStream. Event number was " + ien.getEventNumber()));
+                    " Got stale data from setupAppend on segment " + segmentId + " for ConditionalOutputStream. Flow number was " + ien.getRequestId()));
```

Comment:
Can we log both?

Reply:
Done. Logging both eventNumber and requestId